dvc/utils/__init__.py

```diff
@@ -232,7 +232,7 @@ def _to_chunks_by_chunks_number(list_to_split, num_chunks):
 def to_chunks(list_to_split, num_chunks=None, chunk_size=None):
     if (num_chunks and chunk_size) or (not num_chunks and not chunk_size):
         raise ValueError(
-            "One and only one of 'num_chunks', 'chunk_size' must be defined"
+            "Only one among `num_chunks` or `chunk_size` must be defined."
```

Comment:
For consistency, to match the one other instance:

https://github.com/iterative/dvc/blob/2c003fdc04633f48446d2ae7bf6722fc92b34656/dvc/remote/slow_link_detection.py#L16-L18

Reply:
Actually @efiop what are `num_chunks` and `chunk_size`? These are not arguments or something exposed to the user. Could this message be better or is it just for debugging? Thanks

Reply:
@jorgeorpinel That exception is not meant for the user, it is a defensive exception for us, developers. :slightly_smiling_face:

Reply:
OK thanks for clarifying.

Per https://github.com/iterative/dvc/pull/2644#discussion_r338417698, back tick is correct here.